nautobot/ui/craco.config.js

```diff
@@ -27,10 +27,10 @@ module.exports = {
 
             // Sourcemaps are our friend: https://blog.teamtreehouse.com/introduction-source-maps
             // Prefer a sourcemap file so we can analyze our apps bundle size in dev with `npm run analyze`.
-            webpackConfig.devtool = "eval-source-map";
-            if (!process.env.NAUTOBOT_DEBUG) {
+            webpackConfig.devtool = "hidden-source-map";
+            if (process.env.NAUTOBOT_DEBUG) {
                 // Smaller sourcemap for production and not linked so browsers don't try to fetch it.
```

Comment:
Since you inverted the logic here, do the comments need to be swapped around or otherwise updated?